scripts/zones/Labyrinth_of_Onzozo/mobs/Lord_of_Onzozo.lua

```diff
@@ -6,6 +6,24 @@ mixins = {require("scripts/mixins/rage")}
 require("scripts/globals/regimes")
 -----------------------------------
 
+function onMobInitialize(mob)
+    mob:setMobMod(dsp.mobMod.DRAW_IN, 1)
+end
+
+function onMonsterMagicPrepare(mob, target)
```

Comment:
have to change his pool entry to say he's a scripted caster for this to work (which I think is dumb, but thats how it currently is - if you wanna change that be my guest)

Reply:
Done